components/developer/golang-116/Makefile

```diff
@@ -0,0 +1,128 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2021 Till Wegmueller
+#
+
```

Comment:
Please modernize the Makefile according to doc/current-tasks.md. E.g. put
BUILD_BITS= 64
BUILD_STYLE= justmake
before the include
and then later remove the common tasks:
build:...
install:...
test:..

Reply:
I looked into current-tasks.md and could not find things that apply here.

Applying your suggestions, makes the build fail. I think this is a too heavily modified Makefile that we can modernise it.

Reply:
> I looked into current-tasks.md and could not find things that apply here.
> 
> Applying your suggestions, makes the build fail. I think this is a too heavily modified Makefile that we can modernise it.

I think it could work - it worked for me even with older components' Makefiles. Here it could be more visible: 
https://github.com/OpenIndiana/oi-userland/pull/6543/files
In your case the BUILD_STYLE is justmake. And also, you can remove the lines 
```
include $(WS_MAKE_RULES)/prep.mk
include $(WS_MAKE_RULES)/justmake.mk
include $(WS_MAKE_RULES)/ips.mk
```
and have `include $(WS_MAKE_RULES)/common.mk` instead.

Reply:
Nona is right. It also works on my machine.

Reply:
Well if I knew to include `$(WS_MAKE_RULES)/common.mk` yes then it works :)

New makefile format applied.